dcmgr/lib/dcmgr/drivers/hypervisor/linux_hypervisor.rb

```diff
@@ -60,6 +60,10 @@ def check_interface(hc)
 
           network = hc.rpc.request('hva-collector', 'get_network', vnic[:network_id])
 
+          unless network[:dc_network][:name].nil?
```

Comment:
This isn't quire right. You're raising an error when a dc_network name is set. It should raise an error when dc_network is not set.